### PR TITLE
3.3: Doc Update - simple-by-example - Add warning

### PR DESCRIPTION
**Pull request by vinaypyati**
In reference to arangodb/planning#2280.

### CODE REVIEW

**Comment by Simran-B** [comment]
@dothebart I moved the warning to the RESTDESCRIPTION, which renders fine in the docs:

image

But this is the Swagger UI:

image

`{% ... %}` is rendered verbatim, and we actually don't want inter-documentation links anywhere in there because they won't work.

**Comment by Simran-B** [comment]
@dothebart I got it to render in the docs only and the place where it appears is okay (it was not a conscious decision however - what does actually control the oder?)

image

What would be nice: Stripping the `{% ... %}` stuff and removing links, then append `'\n**Note**:%s\n' % hintStripped` to the description. But it would need to be removed again in `generateMdFiles.md` in order to not have the hint twice in the docs.

**Comment by dothebart** [comment]
@Simran-B  you could probably alter what ends up in the RESTDESCRIPTION in these lines using a regex: https://github.com/arangodb/arangodb/blob/devel/utils/generateSwagger.py#L1361

**Comment by Simran-B** [comment]
@ObiWahn @dothebart Build failed http://jenkins01.arangodb.biz:8080/view/Documentation/job/arangodb-documentation/122/console

```
Error: Not all blocks were found on both sides:
Documentation      |     Programcode:
	program_options_*
Program code:
Generated: @startDocuBlock program_options_*
There is a mismatch of DocuBlocks used in the Documenation the Blocks provided

Error: Not all blocks were found on both sides
```

I don't understand why this occurs. In [generateMdFiles.py](https://github.com/arangodb/oskar/blob/master/containers/documentation.docker/generateMdFiles.py#L939), no blocks should get registered if no Examples/*.json files exist. None are committed in this or the 3.3 branch. They are not generated either. In [3.4](https://github.com/arangodb/arangodb/blob/3.4/utils/generateExamples.sh#L52) they do, but there is [nothing to generate these files in 3.3](https://github.com/arangodb/arangodb/blob/3.3/utils/generateExamples.sh).

The only other related piece of code that comes to mind is this:
https://github.com/arangodb/oskar/blob/master/containers/documentation.docker/buildLib#L593

But if there are no JSON files, it should not do anything right? Or does it come from a stale `/tmp/rawinprog.txt` perhaps?

File: Documentation/Scripts/generateMdFiles.py [hunk 1]
```diff
@@ -146,12 +147,14 @@ def getRestBodyParam():
 def getRestDescription():
     #print >>sys.stderr, "RESTDESCRIPTION"
     if thisVerb['description']:
-        #print >> sys.stderr, thisVerb['description']
-        return RX3[0].sub(RX3[1], thisVerb['description'])
+        description = thisVerb['description']
+        #print >> sys.stderr, description
```
Comment (ObiWahn) [inline Documentation/Scripts/generateMdFiles.py]: use `print(*args, file=sys.stderr)`, `sys.stderr.write()` or better the logging module this does not work in python3 and it has to be ported to oskar

File: Documentation/Scripts/generateMdFiles.py [hunk 2]
```diff
@@ -173,31 +176,31 @@ def validatePathParameters():
         if thisVerb['parameters'][nParam]['in'] == 'path':
             break
     else:
-        raise Exception("@RESTPATHPARAMETERS found without any parameter following in %s " % json.dumps(thisVerb, indent=4, separators=(', ',': '), sort_keys=True))
+        raise Exception("@RESTPATHPARAMETERS found in Swagger data without any parameter following in %s " % json.dumps(thisVerb, indent=4, separators=(', ',': '), sort_keys=True))
```
Comment (ObiWahn) [inline Documentation/Scripts/generateMdFiles.py]: do not use % use .format instead

**Comment by ObiWahn** [comment]
@Simran-B `/tmp` should always be fresh as you should always get a new container. json-files are generated here: https://github.com/arangodb/oskar/blob/master/containers/documentation.docker/entrypoint#L59 for oskar runs.

**Comment by Simran-B** [comment]
Will add something to build script in 3.2 and 3.3 to actively delete Examples/*.json as they might linger around from a previous build after all. If that fixes it, then both PRs are essentially ready.

**Comment by Simran-B** [comment]
New PR with conflicts resolved: https://github.com/arangodb/arangodb/pull/8536

**Pull request closed without merging**